src/elements.rs

```diff
@@ -139,12 +194,33 @@ impl Element {
         let cmd = WebDriverCommand::GetElementProperty(self.element.clone(), prop.to_string());
         match self.client.issue(cmd).await? {
             Json::String(v) => Ok(Some(v)),
+            Json::Bool(b) => {
+                // NOTE: boolean properties such as "checked" seem to be returned as boolean
+                //       values rather than strings.
```

Comment:
That's super interesting. I guess it's because you can have something like `<option selected>`. Though I thought that was shorthand for `<option selected="selected">`. Not quite sure what the right thing to do here is in terms of coercing to a string type. Whatever we do ("true" and "false" are probably okay), we need to document that in the public API, not just in an internal comment.

Reply:
Pretty sure this has caught me out twice. In this case I found it due to the `thirtyfour` doctest failing.
I've moved the comment into the public API docs.